ibm/service/dnsservices/resource_ibm_dns_linked_zone_test.go

```diff
@@ -0,0 +1,211 @@
+	//subnetname := fmt.Sprintf("seczone-subnet-name-%d", acctest.RandIntRange(10, 100))
+	name := fmt.Sprintf("seczone-cr-%s", acctest.RandStringFromCharSet(5, acctest.CharSetAlphaNum))
+	//description := "new test CR - TF"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() {},
+		Providers:    acc.TestAccProviders,
+		CheckDestroy: testAccCheckIBMDNSLinkedZoneDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckIBMDNSLinkedZoneBasic(name),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("ibm_dns_linked_zone.test", "zone", "seczone-terraform-plugin-test.com"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccIBMDNSLinkedZone_update(t *testing.T) {
+	name := fmt.Sprintf("seczone-cr-%s", acctest.RandStringFromCharSet(5, acctest.CharSetAlphaNum))
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() {},
+		Providers:    acc.TestAccProviders,
+		CheckDestroy: testAccCheckIBMDNSLinkedZoneDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckIBMDNSLinkedZoneBasic(name),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("ibm_dns_linked_zone.test", "zone", "seczone-terraform-plugin-test.com"),
+				),
+			},
+			{
+				Config: testAccCheckIBMDNSLinkedZoneUpdateConfig(name, "test description", "label"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("ibm_dns_linked_zone.test", "zone", "seczone-terraform-plugin-test.com"),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckIBMDNSLinkedZoneUpdateConfig(name string, description string, label string) string {
+	return fmt.Sprintf(`
+	resource "ibm_resource_group" "rg" {
+		is_default	= true
+	}
+	resource "ibm_resource_instance" "test-pdns-cr-instance" {
+		name				= "test-pdns-cr-instance"
+		resource_group_id	= data.ibm_resource_group.rg.id
+		service				= "dns-svcs"
+		plan				= "standard"
+	}
+	resource "ibm_dns_linked_zone" "test" {
+		instance_id	= ibm_resource_instance.test-pdns-cr-instance.id
+		zone		= "seczone-terraform-plugin-test.com"
+		description	= "new test CR - TF"
+		label		= "label"
+	}
+	`)
+}
+
+func TestAccIBMDNSLinkedZoneImport(t *testing.T) {
+	var linkedZoneID string
+	name := fmt.Sprintf("seczone-vpc-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { acc.TestAccPreCheck(t) },
+		Providers:    acc.TestAccProviders,
+		CheckDestroy: testAccCheckIBMDNSLinkedZoneDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckIBMDNSLinkedZoneBasic(name),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckIBMDNSLinkedZoneExists("ibm_dns_linked_zone.test", &linkedZoneID),
+					resource.TestCheckResourceAttr("ibm_dns_linked_zone.test", "zone", acc.ISZoneName),
+				),
+			},
+			{
+				ResourceName:      "ibm_dns_linked_zone.test",
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
+					"zone",
```

Comment:
The GET API doesn't give back the field zone?

Reply:
correct ive removed this